CONTRIBUTING-es.md

```diff
@@ -0,0 +1,23 @@
+# Contribuyendo
+
+Por favor, mantenga el [seguidor de problemas](http://github.com/hakimel/reveal.js/issues) limitado a **informes de errores**, **solicitudes de funciones** y **solicitudes de pull**.
```

Comment:
URL should be http://github.com/cubantech/slides.cuban.tech/issues , please change in English version as well. Besides , `seguidor de incidencias` => `gestor de incidencias`